FirstApp/View/HomeViewController.swift

```diff
@@ -33,6 +33,7 @@ class HomeViewController: UIViewController {
         }
         recipeSearchBar.delegate = self
         self.hideKeyboardWhenTappedAround()
+        print("Home: \(username)")
```

Comment:
Same issue as with Log statements, shouldn't be here

Reply:
Do you read my pull request message😅